crates/anvil/core/src/types.rs

```diff
@@ -23,6 +23,10 @@ impl<'de> serde::Deserialize<'de> for Forking {
         #[serde(rename_all = "camelCase")]
         struct ForkOpts {
             pub json_rpc_url: Option<String>,
+            #[serde(
+                default,
+                deserialize_with = "ethers_core::types::serde_helpers::deserialize_stringified_u64_opt"
```

Comment:
unrelated q/comment: we should probably move over some serde helpers from reth once we finish alloyfying foundry, right?

Reply:
from alloy, yes